### DISCUSSION TITLE
【新人报道】萌新绳匠第一次上绳网

### DISCUSSION THREAD

**Discussion by AnbulesPeo** [share121/inter-knot#274] [其他]
你们这个论坛是干什么用的啊【疑惑】

**Comment by Jiesaepe** [comment]
用处可多了，就比如你可以在这里发帖求助，就算是空洞委托，只要报酬足够，也会有绳匠出面帮你解决，就比如前阵子出现的那家伙，直接把绳网上的委托全接了

**Comment by luoyu007** [comment]
楼上说得对，那家伙完成委托的速度特别快，简直就是另一个法厄同
